src/main/java/ninetask/TableSortAndSearch.java

```diff
@@ -0,0 +1,44 @@
+package ninetask;
+
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+
+
+public class TableSortAndSearch {
+
+    public List<Employee> tableSearch(int requiredAge, int requiredSalary) {
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
+        driver.get("https://www.seleniumeasy.com/test/table-sort-search-demo.html");
+        new Select(driver.findElement(By.xpath("//select"))).selectByValue("10");
+        List<WebElement> pages = driver.findElements(By.xpath("//span//a[contains(@class, \"paginate_button \")]"));
+        List<Employee> employees = new ArrayList<>();
+
+        for (int p = 1; p <= pages.size(); p++) {
+            List<WebElement> allRows = driver.findElements(By.xpath("//tbody/tr"));
+
+            for (int i = 1; i <= allRows.size(); i++) {
```

Comment:
Replace with for each (it is possible to call findElement method on WebElement instance)